contracts/lib/LibBalances.sol

```diff
@@ -100,9 +100,9 @@ library Balances {
 
         uint256 liquidationGasCost = liquidationCost * 6;
 
-        uint256 minimumBase = notionalValue / maximumLeverage;
+        uint256 baseMinimumMargin = notionalValue / maximumLeverage;
```

Comment:
Don't know if I'm a fan `baseMinimumMargin`. I get that it is the base minimum amount (and not minimum margin of base). Maybe we could just call this minimumMargin?

Reply:
What about `minimumMarginWithoutGasCost` or something? It's a long variable name but it's only used in 1 other line and aptly describes what it does.

Reply:
yeah I like that @BenjaminPatch